core/src/main/scala/org/broadinstitute/dsde/rawls/config/LeonardoConfig.scala

```diff
@@ -0,0 +1,15 @@
+package org.broadinstitute.dsde.rawls.config
+
+import com.typesafe.config.Config
+
+case class LeonardoConfig(baseUrl: String, wdsType: String)
```

Comment:
this is now unused outside of tests, I believe. You could do away with it entirely, or update `Boot` to use it, like:
```scala
      val leonardoConfig: LeonardoConfig = LeonardoConfig(conf.getConfig("leonardo"))
      val leonardoDAO: LeonardoDAO =
        new HttpLeonardoDAO(leonardoConfig.baseUrl, leonardoConfig.wdsType)
```